Added Space Shooter Game in Demo using Hooman Library

vamsipasam2000 opened the pull request:
How to play the space shooter game: 

- The player's spaceship is positioned at the bottom of the screen and can move left or right using left arrow and right arrow key respectively.
- Asteroids or enemy ships appear from the top and move towards the player's ship.
- The player shoots projectiles by clicking the space bar to destroy incoming objects, earning points for each hit.
- Use the left and right arrow keys to move your spaceship horizontally across the screen.

Abdur-rahmaanJ commented on hooman/demos/spaceshooterB.py:
```diff
@@ -0,0 +1,141 @@
+    # Refresh the screen
+    hapi.flip_display()
+
+pygame.quit()
```
We don't need this in Hooman. Can you please remove it? Thanks!

vamsipasam2000 commented:
@Abdur-rahmaanJ I have added a new .py game file that removed unnecessary pygame.quit call as per review in my latest commit and updated the PR #88 https://github.com/Abdur-rahmaanJ/hooman/pull/88 . Kindly review the changes and merge if everything seems good. Please let me know if any further changes are needed. Thank you!
